resources/sass/abstracts/_mixins.scss

```diff
@@ -0,0 +1,90 @@
+@mixin modal-content {
+    margin-left: 18% !important;
+    //margin-right: 20% !important;
+    width: 63% !important;
+}
+
+@mixin form-left-text {
+    font-size: 16px;
+    font-weight: 500;
+    padding: 3%;
+    font-family: $primary_font;
+    margin-left: 37%;
+    text-align: left;
+}
+
+@mixin form-text-heading {
+    font-weight: 700;
+    font-size: 30px;
+    /* line-height: 50px; */
+    padding: 3%;
+    font-family: $primary_font;
+    margin-left: 37%;
+    text-align: left;
+}
+
+@mixin form-btn {
+    background: $color_white;
+    padding: 5px;
+    width: 100%;
+    height: 35px;
+    border: 1px solid $color_red;
+    margin-top: 40px;
+    cursor: pointer;
+    transition: .3s;
+    box-shadow: 0 50px $color_red inset;
+    color: $color_white;
+}
+
+@mixin div-row {
+    display: flex;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 1140px;
+    //text-align: center
+}
+
+@mixin flex {
```

Comment:
Why you creating these as @mixin they are just going to create duplicate code, make them classes instead and use them wherever you need them